tutorials/02-Basic Operations.ipynb

```diff
@@ -0,0 +1,1822 @@
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$\\displaystyle x^{2} + x + 1$"
+      ],
+      "text/plain": [
+       "x**2 + x + 1"
+      ]
+     },
+     "execution_count": 112,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from sympy.parsing.sympy_parser import (\n",
+    "    standard_transformations,\n",
+    "    convert_xor)\n",
+    "\n",
+    "parse_expr('x^2 + x + 1', transformations=standard_transformations + (convert_xor,))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c0a0da10-9e86-4468-9050-6526f7273b1a",
+   "metadata": {},
+   "source": [
+    "And with `implicit_multiplication` option, you can multiply by juxtaposition."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 113,
+   "id": "fc085ff0-6c30-4e79-9d5a-2aac9241163c",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$\\displaystyle 2 x^{2} + 3 x + 4$"
+      ],
+      "text/plain": [
+       "2*x**2 + 3*x + 4"
+      ]
+     },
+     "execution_count": 113,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from sympy.parsing.sympy_parser import (\n",
+    "    standard_transformations,\n",
+    "    convert_xor, \n",
+    "    implicit_multiplication)\n",
+    "\n",
+    "parse_expr('2x^2 + 3x + 4', transformations=standard_transformations + (convert_xor, implicit_multiplication))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bb0d5965-1e16-45f8-99bf-bebf6518aa3d",
+   "metadata": {},
+   "source": [
+    "However, the cons is that even though parsing allows you to create symbols without declaring,\n",
+    "\n",
+    "you would anyway need symbols later for computing derivatives, solving, ...\n",
+    "(to be introduced)\n",
+    "\n",
+    "For that cases, you have to parse strings again for symbols:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 100,
+   "id": "597ede31-1ceb-4695-b12b-d1d499d71d91",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$\\displaystyle \\cos{\\left(x + y \\right)}$"
+      ],
+      "text/plain": [
+       "cos(x + y)"
+      ]
+     },
+     "execution_count": 100,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "parse_expr('sin(x + y)').diff(parse_expr('x'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "743da5e3-f08f-4f90-8acd-b727134ddb07",
+   "metadata": {},
+   "source": [
+    "## $LaTeX$ parser"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "816450d3-82ba-4522-ade3-f106aad29a6b",
+   "metadata": {},
+   "source": [
+    "We also have experimental $LaTeX$ parser\n",
+    "\n",
+    "https://docs.sympy.org/latest/modules/parsing.html#experimental-mathrm-latex-parsing\n",
+    "\n",
+    "You would need to install `antlr` which is difficult to cover in this tutorial,\n",
```

Comment:
For that reason, I would just completely omit this section. We should only cover things that people can actually try (of course, there are hundreds of SymPy features that we won't be able to cover in the tutorial, but that's fine).